### ISSUE TITLE
Bug - unknown dns provider mythicbeasts- dns challenge 

### ISSUE THREAD

**Issue by dan-tfp** [containous/traefik#6971]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

Built container to be handled by Traefik.

### What did you expect to see?

Successful dns challenge, SSL certificate issued by Letsencrypt using Mythic Beasts DNS API. 

### What did you see instead?

DNS challenge failed, certificate was not issued as expected.

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      2.2.1
Codename:     chevrotin
Go version:   go1.14.2
Built:        2020-04-29T18:02:09Z
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```
--certificatesresolvers.myresolver.acme.dnschallenge=true
--certificatesresolvers.myresolver.acme.dnschallenge.provider=mythicbeasts


```

### If applicable, please paste the log output in DEBUG level (`--log.level=DEBUG` switch)

```
time="2020-06-25T00:25:56Z" level=error msg="Unable to obtain ACME certificate for domains \"<subdomain.fqdn>\": cannot get ACME client unrecognized DNS provider: mythicbeasts"
```
Checking the changelog as below the latest version of go-acme/lego being used is v3.6.0:

https://github.com/containous/traefik/blob/master/CHANGELOG.md

The commit in go-acme/lego which introduces this provider using dns challenge is v3.7.0:

https://github.com/go-acme/lego/pull/1136

Thank you for the brilliant software, when will you be issuing a release with go-acme/lego that includes release v3.7.0?

Many thanks

Dan

**Comment by ldez** [comment]
Hello,

currently Traefik (v2.2.1) use lego v3.6.

mythicbeasts has been added in lego [v3.7](https://github.com/go-acme/lego/releases/tag/v3.7.0)

I updated Traefik to use lego v3.7 (https://github.com/containous/traefik/pull/6792) but after the release of Traefik v2.2.1.

So mythicbeasts will be available in the next path release of Traefik (v2.2.2)

**Comment by dan-tfp** [comment]
Hi ldez,

Thank you for your prompt reply on a weekend, much appreciated.

When is v2.2.2 due to be released?

Many thanks
